### ISSUE TITLE
Enter won't sent a message

### ISSUE THREAD

**Issue by Yura-87** [vincelwt/chatgpt-mac#121]
Can you please add new shorcut (or allow Enter key) for sending the prompt/message? It will be very useful.

**Comment by venky-17** [comment]
i can do that, can u please provide me with more context. I am not clear about this

**Comment by Yura-87** [comment]
Yes, I mean this :) hope the image illustration is good for you…
<img width="444" alt="scr" src="https://github.com/vincelwt/chatgpt-mac/assets/15585200/3d6a462f-1d70-4e99-858a-6a44c2199d3a">

**Comment by bhagirath1312** [comment]
you can use cmd + return key  to sending the message
